vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -60,13 +60,17 @@ A cache issue impacting the first deployment of the `entando-hub-content-bundle`
 
 To clear the cache, select `Administration` from the bottom of the left menu, then `Reload configuration`.
 
-Alternatively, restarting the quickstart-server pod (which contains the Entando App Engine) will also clear the cache, and can be achieved with `ent k delete pod/<YOUR QUICKSTART-SERVER POD>`, e.g. `ent k delete pod/quickstart-server-deployment-5d785b997c-r4sc8`. It will take several minutes for the pod to redeploy after deletion. 
+Alternatively, restarting the quickstart-server pod (which contains the Entando App Engine) will also clear the cache, and can be achieved with `ent k delete pod/YOUR-QUICKSTART-POD-NAME`, e.g. `ent k delete pod/quickstart-deployment-5d785b997c-r4sc8`. It will take several minutes for the pod to redeploy after deletion. 
 :::
 
 5. Setup permissions to configure the service:
    - [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin.
    - [Assign client roles](../../docs/consume/identity-management.md#authorization) to authorize permissions. Select `entandopsdh-entando-hub-catalog-server` from `Client Roles` and `eh-admin` from `Available Roles`.
```

Comment:
I think we need to rework this a bit. The 2nd bullet is to give a user the ability to manage the Hub. Maybe something like:
* [Give at least one user the ability](..link) to manage the Hub by granting the `eh-admin` role. Assign the `eh-admin` role for the `entandopsdh-entando-hub-catalog-server`  client.
* Give the generated plugin client the permission to manage users.
**Use the existing bullets.

Reply:
I don't know how to deal with the fact that the 6.x client id is entandopsdh-entando-hub-catalog-server but the 7.x client id is entandopsdh-entando-hub-catalog-ms-server. We can discuss on the call.

Reply:
Maybe we just correct the id for 7.x...